modules/olm-mirroring-catalog-airgapped.adoc

```diff
@@ -83,6 +83,24 @@ $ oc adm catalog mirror \
 Red Hat Quay does not support nested repositories. As a result, running the `oc adm catalog mirror` command will fail with a `401` unauthorized error. As a workaround, you can use the `--max-components=2` option when running the `oc adm catalog mirror` command to disable the creation of nested repositories. For more information on this workaround, see the link:https://access.redhat.com/solutions/5440741[Unauthorized error thrown while using catalog mirror command with Quay registry] Knowledgebase Solution.
 ====
 
+. Run the `oc adm catalog mirror` command again. Use the newly mirrored index image as the source and the same mirror registry namespace used in the previous step as the target:
++
+[source,terminal]
+----
+$ oc adm catalog mirror \
+    <mirror_registry>:<port>/<index_image> \
+    <mirror_registry>:<port>/<namespace> \
+    --manifests-only \//<1>
+    [-a ${REG_CREDS}] \
+    [--insecure]
+----
+<1> The `--manifests-only` flag is required for this step so that the command does not copy all of the mirrored content again.
++
+[IMPORTANT]
+====
+This step is required because the image mappings in the `imageContentSourcePolicy.yaml` file generated during the previous step must be updated from local paths to valid mirror locations. Failure to do so will cause errors when you create the `ImageContentSourcePolicy` object in a later step.
```

Comment:
@adellape is the first "i" UNcapitalized in the yaml filename while the second one (object name) is?

Reply:
(Alex answered this question directly...it's a result of the mirroring tool.)